addons/vehiclelock/CfgEventHandlers.hpp

```diff
@@ -33,4 +33,9 @@ class Extended_InitPost_EventHandlers {
             serverInit = QUOTE(_this call FUNC(handleVehicleInitPost));
         };
     };
+    class Plane {
+        class ADDON {
+            serverInit = QUOTE(_this call FUNC(handleVehicleInitPost));
```

Comment:
`QFUNC(handleVehicleInitPost)`?

Reply:
That would break it, because it removes the `call`.

Reply:
oops... too much time with CBA state machine